lib/nonce.js

```diff
@@ -0,0 +1,37 @@
+const CSP_HEADER = 'Content-Security-Policy';
+const CSPRO_HEADER = 'Content-Security-Policy-Report-Only';
+const SCRIPT_SRC_STRING_LENGTH = 11;
```

Comment:
This is because of `"script-src "`? Maybe I would do `const SCRIPT_SRC_STRING = 'script-src ';` and then you'll have both that and `SCRIPT_SRC_STRING.length` as constants?